src/client/index.ts

```diff
@@ -184,8 +403,9 @@ export async function clear(id: string): Promise<boolean> {
  * ```ts
  * import {getAll} from 'openfin-notifications'
  *
- * getAll()
- *  .then(console.log);
+ * getAll().then((notifications: Notification[]) => {
+ *     console.log(`Service has ${notifications.length} notifications for this app:`, notifications);
+ * });
```

Comment:
Maybe have a specific mention that it is not possible to get notes for a different application?